scipy/integrate/tests/test_ivp.py

```diff
@@ -791,3 +791,86 @@ def structure(n):
                     rtol=1e-12, atol=1e-14)
     assert_allclose(factor_dense, factor_sparse, rtol=1e-12, atol=1e-14)
 
+
+def test_args():
+
+    # sys3 is actually two decoupled systems. (x, y) form a
+    # linear oscillator, while z is a nonlinear first order
+    # system with equilibria at z=0 and z=1.  If k > 0, z=1
+    # is stable and z=0 is ubstable.
+
+    def sys3(t, w, omega, k, zfinal):
+        x, y, z = w
+        return [-omega*y, omega*x, k*z*(1 - z)]
+
+    def sys3_jac(t, w, omega, k, zfinal):
+        print(t)
```

Comment:
Not intentional, right?

Reply:
Oops.  That was meant to be temporary.  I used it to verify that the Jacobian function was being called a few times.  It will be gone in the next update.

Reply:
Fixed.